src/runtime_src/core/tools/common/XBUtilities.h

```diff
@@ -22,16 +23,37 @@
 #include "core/common/device.h"
 #include "core/common/query_requests.h"
 
+#include <chrono>
+#include <iostream>
 #include <string>
 #include <memory>
 #include <map>
-#include <iostream>
 #include <vector>
-#include <boost/property_tree/ptree.hpp>
-#include <boost/program_options.hpp>
+
 #include <boost/algorithm/string.hpp>
+#include <boost/program_options.hpp>
+#include <boost/property_tree/ptree.hpp>
 
 namespace XBUtilities {
+
+  class Timer {
+  private:
+    std::chrono::high_resolution_clock::time_point m_time_start;
+
+  public:
+    Timer() { reset(); }
+
+    std::chrono::duration<double> get_time() 
+    {
+      std::chrono::high_resolution_clock::time_point time_end = std::chrono::high_resolution_clock::now();
+      return std::chrono::duration<double>(time_end - m_time_start);
+    }
+
+    void reset() { m_time_start = std::chrono::high_resolution_clock::now(); }
```

Comment:
Function name too vague.

Reply:
What would you think the name should be? A 'Reset' function for a timer seems clear to me.